modules/ROOT/pages/design-async-api.adoc

```diff
@@ -236,8 +236,90 @@ components:
           email:
             type: string
 ----
+== Example AsyncAPI 2.6.0 Specification (Inline Avro)
 
-== Example AsyncAPI 2.0 specification
+If you created an AsyncAPI specification project, you can replace the initial API specification with following example code to test inlining the API specification fragments:
+
+
+[source,yaml]
+----
+asyncapi: 2.6.0
+info:
+  title: Example AsyncAPI Spec (Inline Avro)
+  version: '1.0.0'
+channels:
+  myChannel:
+    publish:
+      message:
+        schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
+        payload:
+          name: AmericanFlightDataType
+          type: record
+          fields:
+            - name: ID
+              type: int
+              example: 1
```

Comment:
"example" entry in all these fields are not defined by the spec, after discussing with the team, a good alternative could be "default", although it has a different purpose it fills the console & triggers the validation if the value is incorrect

- name: ID
   type: int
   default: 1

Can you replace them? the same applies for line 264, 267 & so on